templates/web/base/tokens/confirm_update.html

```diff
@@ -1,24 +1,18 @@
 [% INCLUDE 'header.html', bodyclass = 'fullwidthpage', title => loc('Confirmation') %]
 
 <div class="confirmation-header">
-  <h1>[% loc('Your update has been confirmed!') %]</h1>
 
-<p class="confirmed">
-[% IF creator_fixed %]
-[%
-    tprintf(loc('Thank you &mdash; you can <a href="%s">view your updated problem</a> on the site.'),
-        c.uri_for( '/report', problem_id ) );
-%]
-[% ELSE %]
-[%
-    tprintf( loc('You have successfully confirmed your update and you can now <a href="%s">view it on the site</a>.'),
-        c.uri_for( '/report', update.problem.id ) _ '#update_' _ update.id );
-%]
-[% END %]
-</p>
+  <h1><a href="[% c.cobrand.base_url_for_report( update.problem ) %][% update.problem.url %]">[% update.problem.title %]</a></h1>
```

Comment:
This URL doesn't match the URL of he update – I said above this was presumably wrong, but looking at i again perhaps it was deliberate? Note that it used to link differently depending upon if the creator had just marked it as fixed during an update.